naming/src/main/java/com/alibaba/nacos/naming/core/v2/service/impl/PersistentClientOperationServiceImpl.java

```diff
@@ -170,7 +171,10 @@ public Response onApply(WriteRequest request) {
     
     private void onInstanceRegister(Service service, Instance instance, String clientId) {
         Service singleton = ServiceManager.getInstance().getSingleton(service);
-        Client client = clientManager.computeIfAbsent(clientId, () -> new IpPortBasedClient(clientId, false));
+        if (!clientManager.contains(clientId)) {
```

Comment:
This `contains` is unnecessary.

Reply:
To reduce invalid object create. When the node handle the register persistent instance, the client has been create and register. There is no need to create and try create again.